README.md

```
@@ -1,11 +1,23 @@
-# veto
+<p align="center">
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)